synedrion/src/curve/arithmetic.rs

```diff
@@ -1,132 +1,169 @@
 use alloc::{format, string::String, vec, vec::Vec};
-use core::{
-    default::Default,
-    ops::{Add, Mul, Neg, Sub},
-};
+use core::ops::{Add, Mul, Neg, Rem, Sub};
+#[cfg(test)]
```

Comment:
That'll only apply to the first `use` statement.

Reply:
Yes, that's on purpose, so that `TestParams` can be exported.